frontend/src/metabase/components/Timeline/Timeline.tsx

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import _ from "underscore";
+import { t } from "ttag";
+import { getRelativeTime } from "metabase/lib/time";
+
+import type { Revision } from "metabase-types/api";
+import Button from "metabase/core/components/Button";
+import Tooltip from "metabase/core/components/Tooltip";
+
+import {
+  TimelineContainer,
+  TimelineEvent,
+  Border,
+  StyledIcon,
+  EventBody,
+  EventHeader,
+  Timestamp,
+} from "./Timeline.styled";
+
+const ICON_SIZE = 16;
```

Comment:
We usually use `rem` for spacing and not pixels.

Another thing is that it looks like something styles-specific, so we could move these constants to styled components and use them there, and remove props like `leftShift`

Reply:
Yeah I think this is leftover from the old component implementation, I will remove the props